CliWrap/Command.cs

```diff
@@ -331,10 +331,16 @@ private ProcessStartInfo GetStartInfo()
             RedirectStandardInput = true,
             RedirectStandardOutput = true,
             RedirectStandardError = true,
-            UseShellExecute = false,
-            CreateNoWindow = true
+            UseShellExecute = false
         };
 
+        // Setting CreateNoWindow has a 30ms overhead added to execution time of the process.
+        // A window won't be created for console applications even when CreateNoWindow = false,
+        // so it's only necessary to set it if there is no console.
+        // This check is only necessary on windows platforms because CreateNoWindow does not work on MacOS or Linux.
```

Comment:
```suggestion
        // This check is only necessary on Windows platforms because CreateNoWindow does not work on MacOS or Linux.
        // https://github.com/Tyrrrz/CliWrap/pull/142
```